src/views/Library.vue

```diff
@@ -12,11 +12,18 @@
     </VRow>
     <VRow v-if="rescan">
       <VBtn @click="start" color="success" class="ma-2">
```

Comment:
Maybe disable the button? (Using the same condition as the spinning icon.)
You probably want to extract the condition to a computed property in that case.